scapy/contrib/cansocket_native.py

```diff
@@ -71,6 +74,17 @@ def __init__(self,
                 "Could not modify receive own messages (%s)", exception
             )
 
+        if self.fd:
+            try:
+                self.ins.setsockopt(socket.SOL_CAN_RAW,
+                                    socket.CAN_RAW_FD_FRAMES, 
+                                    1)
+                self.MTU = CAN_FD_MTU
+            except Exception as exception:
```

Comment:
very broad catch. Please specify a Exception type or a set of exceptions

Reply:
thats the same exception used in setsockopt for default CAN socket?